packages/vitest/src/node/cli.ts

```diff
@@ -16,21 +16,21 @@ cli
   .option('-w, --watch', 'watch mode')
   .option('-t, --testNamePattern <pattern>', 'run test names with the specified pattern')
   .option('--ui', 'enable UI')
-  .option('--open', 'open UI automatically', { default: true })
+  .option('--open', 'open UI automatically')
```

Comment:
This will remove `(default: true)` text from `vitest --help` output. We can add this to option description.

Reply:
I'll add them